packages/server/src/server.ts

```diff
@@ -410,7 +410,8 @@ app.useRpc('eth_getLogs', async (params: any) => {
     params?.toBlock || null,
     params?.address || null,
     params?.topics || null,
-    requestId
+    requestId,
+    params?.limitParams || undefined
```

Comment:
Per the specifications, `eth_getLogs` does not accept a `limitParams` argument,  so I don't think we should include custom parameters to the Filter object (at this point params value is overwritten with params[0] which is the filter object). And I think this should be internal for the `getLogs` function in the `eth` module.

Reply:
Good point. Agreed, we should be exposing a new prop. Rather we should just always default to ask for the max the mirror node will allow for to reduce the number of trips

Reply:
This is optional parameter and it's needed for testing. Otherwise we have to log more than 100 logs per test. This way, I'm limiting it to 2 (for example) and test the functionality easy.

Reply:
Changing the specifications or the code so you can ease testing seems as a wrong approach. This feels more like adjusting the code to the test case. At best it should be an env var or the default, and in the tests the module should be reinitialized with the env var set to a specific number for this specific test.

Reply:
Actually don't you just have to simulate that there's a link in the response in the mock? It shouldn't matter how many logs you return as long as there is a link for the next query.

Reply:
Yeah. Agreed with Maksim, add an env variable for default log results request amount. We can default to 100, in tests you can set it smaller. This also allows for easy modification should mirror node modify the limit

Reply:
Done. Added `.env` variable